pkg/serial/handshake.go

```diff
@@ -32,10 +32,10 @@ import (
 )
 
 const (
-	SYN       = 0x16
-	ACK       = 0x06
-	DEBUG_ACK = 0x07
-	NAK       = 0x15
+	CodeSync     = 0x16
```

Comment:
Doesn't look like these constants are referenced outside the package, so let's not export them either (syn, ack, debugAck, nak).  Or if we want to prefix, maybe use 'flag' or 'seq'?  seqSyn, seqAck, etc